Dockerfile

```diff
@@ -1,29 +1,47 @@
-FROM kbase/sdkbase2 as build
+FROM eclipse-temurin:11-jdk as build
 
 COPY . /tmp/workspace_deluxe
-RUN pip install configobj && \
-    cd /tmp && \
+WORKDIR /tmp
+RUN apt-get update -y && \
+    apt-get install -y ant git ca-certificates python3-sphinx && \
     git clone https://github.com/kbase/jars && \
     cd workspace_deluxe && \
     make docker_deps
 
-FROM kbase/kb_jre
+# updated/slimmed down version of what's in kbase/kb_jre
+FROM ubuntu:18.04
```

Comment:
Shouldn't this be from some specific jre image for java 11?

Reply:
My particular concern is that it's not clear what version of java is being run and whether that will change as the ubuntu image is updated (my understanding it that it's not supposed to but...). Are you sure that java 11 is what's being run here?

If so and you're going to update the image to be more explicit about the java version being run I think that's ok

Reply:
```sh
❯ docker build . -t workspace_image
[...]
❯ docker run -it workspace_image bash
```
```bash
root@aeb6bbc85848:/kb/deployment/services/workspace/tomcat# which java 
/usr/bin/java
root@aeb6bbc85848:/kb/deployment/services/workspace/tomcat# java --version
openjdk 11.0.17 2022-10-18
OpenJDK Runtime Environment (build 11.0.17+8-post-Ubuntu-1ubuntu218.04)
OpenJDK 64-Bit Server VM (build 11.0.17+8-post-Ubuntu-1ubuntu218.04, mixed mode, sharing)
root@aeb6bbc85848:/kb/deployment/services/workspace/tomcat# update-alternatives --config java
There is only one alternative in link group java (providing /usr/bin/java): /usr/lib/jvm/java-11-openjdk-amd64/bin/java
Nothing to configure.
```

Reply:
...and I do plan to update the image to be explicit about the version of java installed.